test/parallel/test-whatwg-url-properties.js

```diff
@@ -25,6 +25,9 @@ const expected = ['toString',
                   'username', 'password', 'host', 'hostname', 'port',
                   'pathname', 'search', 'searchParams', 'hash', 'toJSON'];
 
+const obj = { toString() { throw new Error('toString'); } };
```

Comment:
I think you have already added non-WPT tests to that file? Most error message validation tests are placed this way(after WPT) though.